docs/docs/reference/command-line-interface.md

```diff
@@ -952,6 +955,9 @@ meltano --environment=dev run --force tap-gitlab target-postgres tap-salesforce
 # run a pipeline with a custom state ID suffix
 # the autogenerated ID for the EL pair will be 'dev:tap-gitlab-to-target-postgres:pipeline-alias'
 meltano --environment=dev --state-id-suffix pipeline-alias run tap-gitlab hide-secrets target-postgres
+
+# run the pipelines in series, performing a merging of state at the end of run.
+meltano --environment=dev run --merge-state tap-gitlab target-postgres tap-salesforce target-mysql ...
```

Comment:
This two features mentioned in tandem may lead to confusion, e.g. "is it gonna merge state from the two pipelines together?".

A simpler example would be more clear:

```suggestion
# run a pipeline, merging state with that of previous runs.
meltano --environment=dev run --merge-state tap-gitlab target-postgres
```